site/src/components/Expander/Expander.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from "@material-ui/core/Link"
+import makeStyles from "@material-ui/core/styles/makeStyles"
+import { CloseDropdown, OpenDropdown } from "components/DropdownArrows/DropdownArrows"
+
+const Language = {
+  expand: "More",
+  collapse: "Less",
+}
+
+export interface ExpanderProps {
+  expanded: boolean
+  setExpanded: (val: boolean) => void
+}
+
+export const Expander: React.FC<ExpanderProps> = ({ expanded, setExpanded }) => {
+  const toggleExpanded = () => setExpanded(!expanded)
+  const styles = useStyles()
+  return (
+    <Link
+      aria-expanded={expanded}
+      onClick={toggleExpanded}
+      className={styles.expandLink}
+      tabIndex={0}
```

Comment:
We shouldn't need a tabIndex here because links are inherently selectable. Was there a reason you added it?

Reply:
Just copied it from `ErrorSummary`! I'll remove it.